### PR TITLE
Clean up and de-dup the docs repository

### PR DESCRIPTION
**Pull request by bensigelman**
Larger changes:
- Make the "conventions and terminology" doc mostly a pointer to the formal spec
- Make the "data conventions" doc mostly a pointer to the spec's respective yaml file

Smaller changes: basically just copy-editing of other files that I had time for.

### CODE REVIEW

File: _docs/pages/spec.md [hunk 1]
```diff
@@ -1,185 +1,10 @@
...
-
-The per-platform OpenTracing API libraries *may* provide support for configuring (Go: `InitGlobalTracer()`, py: `opentracing.tracer = myTracer`) and retrieving (Go: `GlobalTracer()`, py: `opentracing.tracer`) a global/singleton Tracer instance. If a global Tracer is supported, the default must be the no-op Tracer.
+Both are versioned and the github repository is tagged according to the rules described at [the top of the specification document](https://github.com/opentracing/specification/blob/master/specification.md#versioning-policy).
```
Comment (yurishkuro) [inline _docs/pages/spec.md]: maybe `s/at the top of the specification document/by the versioning policy/`

File: _docs/pages/supported-tracers.md [hunk 2]
```diff
@@ -6,30 +6,12 @@
 
 ## Jaeger
 
-[Jaeger \ˈyā-gər\](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
+[Jaeger \ˈyā-gər\ˈ](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
```
Comment (yurishkuro) [inline _docs/pages/supported-tracers.md]: why the second `'`? The first one denotes the accented syllable.

File: _docs/pages/spec.md (hunk 1 above)
Comment (bensigelman) [inline _docs/pages/spec.md]: good idea, yep... I didn't have the link there when I wrote the original copy.

File: _docs/pages/supported-tracers.md (hunk 2 above)
Comment (bensigelman) [inline _docs/pages/supported-tracers.md]: ah -- I was fixing the fact that there was an orphan `\` character by guessing what was supposed to be escaped :)  Removed.

File: _docs/pages/supported-tracers.md [hunk 3]
```diff
@@ -6,30 +6,12 @@
 
 ## Jaeger
 
-[Jaeger \ˈyā-gər\](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
+[Jaeger \ˈyā-gər](http://uber.github.io/jaeger) is Uber's distributed tracing system, built with OpenTracing support from inception. Jaeger includes OpenTracing client libraries in several languages: [Java](https://github.com/uber/jaeger-client-java), [Go](https://github.com/uber/jaeger-client-go), [Python](https://github.com/uber/jaeger-client-python), and [Node.js](https://github.com/uber/jaeger-client-node).
```
Comment (yurishkuro) [inline _docs/pages/supported-tracers.md]: image

Comment (bensigelman) [inline _docs/pages/supported-tracers.md]: @yurishkuro ah :)

Ok, look again. My local `.md` syntax highlighter is more pedantic than jekyll about the escaping, it seems. What I have in the latest revision pleases both the highlighter and jekyll.

**Pull request merged**